domain/src/main/java/org/fao/geonet/domain/SchematronCriteriaType.java

```diff
@@ -52,6 +52,12 @@
         public boolean accepts(ApplicationContext applicationContext, String value, int metadataId, Element metadata,
                                List<Namespace> metadataNamespaces) {
 
+            // if values is empty, consider the criteria as not accepted.
+            // Since the value is editable in the schematron admin interface
+            // we cannot rely on this user input.
+            if (value == null || value.equals("")) {
```

Comment:
`StringUtils.isEmpty()` if available (might not in `domain` module, either provided by spring-util or apache-commons)